### ISSUE TITLE
3-Panel View

### ISSUE THREAD

**Issue by farmaazon** [enso-org/enso#14629]
In scope of this task:
1. Make Drive always visible, as one of the three panels.
2. Tabs are now part of the middle "Graph Editor" panel
3. There should be a focus indicator which of the three panels is currently focused (selection may have less vivid color)
4. The "top bar" with now "user bar" only may stay. 
5. All panels should be resizable and hide-able by toggle button.
